src/containers/Admin/EditTab/HeadingAndDistance/HeadingAndDistance.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+    getFromLocalStorage,
+    saveToLocalStorage,
+} from '../../../../settings/LocalStorage'
+import { useDebounce } from '../../../../hooks/useDebounce'
+import classes from './HeadingAndDistance.module.scss'
+
+const HeadingAndDistance: React.FC = () => {
+    const [settings, setSettings] = useSettings()
+
+    const [distance, setDistance] = useState<number>(settings.distance)
+
+    const debouncedDistance = useDebounce(distance, 800)
+
+    useEffect(() => {
+        if (settings.distance !== debouncedDistance) {
+            setSettings({
+                distance: debouncedDistance,
+            })
+        }
+    }, [debouncedDistance, setSettings, settings])
+
+    const validateInput = useCallback((e: SyntheticEvent<HTMLInputElement>) => {
+        const newDistance = Number(e.currentTarget.value)
+        if (1 <= newDistance && 1000 >= newDistance) {
+            setDistance(newDistance)
+        } else if (newDistance < 1) {
+            setDistance(1)
+        } else {
+            setDistance(1000)
+        }
+    }, [])
+
+    const [showTooltip, setShowTooltip] = useState<boolean>(false)
+
+    useEffect(() => {
+        if (!getFromLocalStorage('hasShownTooltip')) {
+            setShowTooltip(true)
+            saveToLocalStorage('hasShownTooltip', true)
+        }
+    }, [])
+
+    return (
+        <div>
+            <Heading2 className={classes.Heading}>
+                Viser kollektivtilbud innenfor
+                <div className={classes.InputWrapper}>
+                    <Tooltip
+                        content={
+                            <div className={classes.TooltipContainer}>
+                                <Heading4 className={classes.TooltipText}>
+                                    Endre på avstanden?
+                                </Heading4>
+                                <SubParagraph className={classes.TooltipText}>
+                                    Klikk på tallet for å skrive en ny verdi.
+                                </SubParagraph>
+                            </div>
+                        }
+                        placement="bottom"
+                        isOpen={showTooltip}
+                        showCloseButton={true}
+                        disableHoverListener={true}
+                        disableFocusListener={true}
+                    >
+                        <TextField
+                            label=""
+                            className={classes.ExpandingTextField}
+                            size="medium"
+                            defaultValue={distance}
+                            onChange={validateInput}
+                            append="m"
+                            type="number"
+                            max={1000}
+                            min={1}
+                            maxLength={4}
+                            minLength={1}
+                        />
+                    </Tooltip>
+                </div>
```

Comment:
Ser ut som denne har mistet noe styling ila. overgangen 
![image](https://user-images.githubusercontent.com/18345134/206443498-49fb0412-3a83-4356-a9c9-aeb71bc0d0f5.png)

Reply:
Ser ut som den blir overwritten av styling i EditTab

Reply:
Tanker om å ikke bruke entur component her? Virker som vi gjør mye for å jobbe rundt det

Reply:
Så rart! Det så riktig ut da jeg commit'et, men nå ser jeg at det ser ut som på bilde ditt 🤔 Tror nok det er lurest å gjøre som du sier og bytte den ut med en <input>.